.github/workflows/ci.yml

```diff
@@ -11,11 +11,13 @@ jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         ghc:
-          - "9.0.2"
-          - "9.2.4"
-          - "9.4.8"
+          - "9.0.1"
```

Comment:
I think we should also do the last version of each branch, but we can do that separately.

Reply:
We get the latest release covered in the Nix builds. Nix tends to not keep all the releases around, so I test the newer ones via Nix and the oldest via Cabal directly.

It does seem like the gold tests are passing in 9.4.8 (on Linux), but not in 9.4.1.